### PR TITLE
fix-provider-crash-46

### PR DESCRIPTION
**Pull request by tuvia-akeyless**
panic when updating role with some wrong configuration

### CODE REVIEW

File: akeyless/resource_role.go [hunk 1]
```diff
@@ -64,6 +64,16 @@ func resourceRole() *schema.Resource {
 							Description: "Treat sub claims as case-sensitive",
 							Default:     "true",
 						},
+						"access_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "",
+						},
+						"id": {
```
Comment (renanaAkeyless) [inline akeyless/resource_role.go:72]: what is id stand for?

File: akeyless/resource_role.go [hunk 2]
```diff
@@ -64,6 +64,16 @@ func resourceRole() *schema.Resource {
...
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "",
```
Comment (renanaAkeyless) [inline akeyless/resource_role.go:75]: Description ? ( please take from cli )

File: akeyless/resource_role.go (hunk 1 above)
Comment (tuvia-akeyless) [reply akeyless/resource_role.go:72]: it is the assoc-id. we must store it for the deletion.
the delete API require only id...

File: akeyless/resource_role.go [hunk 3]
```diff
@@ -64,6 +64,16 @@ func resourceRole() *schema.Resource {
 							Description: "Treat sub claims as case-sensitive",
 							Default:     "true",
 						},
+						"access_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "",
```
[the remark this answers is not in the document]
Comment (tuvia-akeyless) [reply akeyless/resource_role.go:70]: we don't add description for computed params. it is internal.
see example in: akeyless/data_source_rsa_public.go

File: akeyless/resource_role.go (hunk 2 above)
Comment (tuvia-akeyless) [reply akeyless/resource_role.go:75]: same

File: akeyless/resource_role.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (renanaAkeyless) [reply akeyless/resource_role.go:70]: we need to... so please add Description

File: akeyless/resource_role.go (hunk 2 above)
Comment (renanaAkeyless) [reply akeyless/resource_role.go:75]: same

File: akeyless/resource_role.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (tuvia-akeyless) [reply akeyless/resource_role.go]: V
if need, i can open other PR to fill Description in other resources (probably low priority)

File: akeyless/resource_role.go (hunk 2 above)
Comment (tuvia-akeyless) [reply akeyless/resource_role.go]: V

**Review by renanaAkeyless — APPROVED** [review]

**Pull request merged**